chef/cookbooks/crowbar/templates/default/crowbar-pre-upgrade.sh.erb

```diff
@@ -92,25 +92,25 @@ zypper --non-interactive up <%= @neutron_agent %>
 crudini --set /etc/neutron/neutron-openvswitch-agent.conf.d/200-crowbar-upgrade.conf ovs ovsdb_interface vsctl
 crudini --set /etc/neutron/neutron-openvswitch-agent.conf.d/200-crowbar-upgrade.conf ovs of_interface ovs-ofctl
 <% end %>
-systemctl start <%= @neutron_agent %>
+systemctl restart <%= @neutron_agent %>
```

Comment:
> Is it correct to deploy neutron-network role to compute-ha node

In general not. But there will be at least an l2-agent running on the compute node. In the DVR case there will also be the l3-agent. (In the compute node-HA case I assume those will be managed by pacemaker, but you're right, we have not looked at compute node HA for upgrades yet)